addons/mail/models/mail_thread.py

```diff
@@ -4123,30 +4132,61 @@ def _truncate_payload(self, payload):
     def message_subscribe(self, partner_ids=None, subtype_ids=None):
```

Comment:
a bit of explanation about partner_ids will be nice in the doc bellow. It's not intuitive that it accepts a dictionary or a list